hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -856,6 +864,10 @@ message DeletedKeys {
     repeated string keys = 3;
 }
 
+message DeleteKeysResponse {
```

Comment:
Usually we have a XXXRequest match with XXXResponse. Here we have DeleteKeysRequest->DeleteKeysResponse and UnDeletedKeysResponse. 

NIT: can we keep this consistent to avoid confusion for UnDeleteKeysRequest if any in the future?

```suggestion
message DeleteKeysResponse {
    repeated KeyInfo deletedKeys = 1;
    repeated KeyInfo unDeletedKeys = 2;
}```